stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/repository/ReactionRepository.kt

```diff
@@ -16,6 +18,7 @@ internal class ReactionRepository(private val reactionDao: ReactionDao) {
         insert(listOf(reaction.toEntity()))
     }
 
+    @VisibleForTesting
     internal suspend fun insert(reactions: List<Reaction>) {
```

Comment:
Are we going to remove that at some point?

Reply:
Yes, since there is no point to insert a list directly. Now standard mechanism of inserting reactions is inserting in a transaction with a message insertion. Only single direct reaction makes sense (when you make a new reaction and update offline storage)